pay-api/src/pay_api/models/routing_slip.py

```diff
@@ -195,9 +195,7 @@ def _add_receipt_number(cls, query, search_filter):
     def _add_folio_filter(cls, query, search_filter):
         if folio_number := search_filter.get('folioNumber', None):
             query = query.join(Invoice).filter(
-                and_(Invoice.routing_slip == RoutingSlip.number, and_(Invoice.folio_number == folio_number,
-                                                                      Invoice.payment_method_code.in_(
-                                                                          [PaymentMethod.INTERNAL.value]))))
+                and_(Invoice.routing_slip == RoutingSlip.number, Invoice.folio_number == folio_number))
```

Comment:
do we need to map, businessIdentifier field here, in frontend we are showing businessIdentifier field frominvoice instead of folionumber